aiplatform/text-embeddings/embeddings.go

```diff
@@ -14,7 +14,7 @@
 
 package snippets
 
-// [START aiplatform_text_embeddings]
```

Comment:
I think these might be unintended changes in your PR. please revert this if so.

these region tags are used in existing docs pages, and should not be changed without coordinating changes to the docs.

Reply:
reverted